spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -83,70 +84,128 @@
+          before do
+            allow(user).to receive(:establishing_in_person_enrollment).and_return(nil)
+          end
+
+          it 'disables double address verification for the user' do
+            expect_any_instance_of(Idv::Agent).to receive(:proof_resolution).
+              with(
+                kind_of(DocumentCaptureSession),
+                should_proof_state_id: anything,
+                trace_id: subject.send(:amzn_trace_id),
+                threatmetrix_session_id: nil,
+                user_id: anything,
+                request_ip: request.remote_ip,
+                double_address_verification: false,
+              )
+
+            put :update
+          end
+        end
 
-        put :update
-      end
+        it 'passes the X-Amzn-Trace-Id to the proofer' do
+          expect_any_instance_of(Idv::Agent).to receive(:proof_resolution).
+            with(
+              kind_of(DocumentCaptureSession),
+              should_proof_state_id: anything,
+              trace_id: subject.send(:amzn_trace_id),
+              threatmetrix_session_id: nil,
+              user_id: anything,
+              request_ip: request.remote_ip,
+              double_address_verification: false,
+            )
 
-      it 'only enqueues a job once' do
-        put :update
-        expect_any_instance_of(Idv::Agent).to_not receive(:proof_resolution)
+          put :update
+        end
 
-        put :update
-      end
+        it 'only enqueues a job once' do
+          put :update
+          expect_any_instance_of(Idv::Agent).to_not receive(:proof_resolution)
 
-      context 'when pii_from_user is blank' do
-        it 'redirects' do
-          flow_session[:pii_from_user] = {}
           put :update
-          expect(response.status).to eq 302
         end
-      end
 
-      context 'when different users use the same SSN within the same timeframe' do
-        let(:user2) { create(:user) }
+        context 'when pii_from_user is blank' do
+          it 'redirects' do
+            flow_session[:pii_from_user] = {}
+            put :update
+            expect(response.status).to eq 302
+          end
+        end
 
-        before do
-          allow(IdentityConfig.store).to receive(:proof_ssn_max_attempts).and_return(3)
-          allow(IdentityConfig.store).to receive(:proof_ssn_max_attempt_window_in_minutes).
-            and_return(10)
+        context 'when different users use the same SSN within the same timeframe' do
+          let(:user2) { create(:user) }
+
+          before do
+            allow(IdentityConfig.store).to receive(:proof_ssn_max_attempts).and_return(3)
+            allow(IdentityConfig.store).to receive(:proof_ssn_max_attempt_window_in_minutes).
+              and_return(10)
+          end
+
+          it 'throttles them all' do
+            put :update
+            #          expect_any_instance_of(Idv::Agent).to receive(:proof_resolution)
```

Comment:
I see these commented out lines existed before, but could we remove them? what are they helping with?